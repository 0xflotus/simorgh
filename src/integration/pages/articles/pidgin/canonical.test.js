/**
 * @service pidgin
 * @pathname /pidgin/articles/cwl08rd38l6o
 */

import runCrossPlatformTests from '../crossPlatformTests';
import runCanonicalTests from '../canonicalTests';

describe('Canonical', () => {
  describe(pageType, () => {
    runCrossPlatformTests();
    runCanonicalTests();
  });
});
