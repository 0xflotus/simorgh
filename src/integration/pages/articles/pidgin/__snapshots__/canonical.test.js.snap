// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Canonical Articles A11y Banner should contain product and localised service name  1`] = `"BBC News, Pidgin"`;

exports[`Canonical Articles A11y Banner should contain product and localised service name with lang 1`] = `"en-GB"`;

exports[`Canonical Articles A11y I can see the skip to content link 1`] = `"Waka go wetin de inside"`;

exports[`Canonical Articles Analytics ATI: https://logws1363.ati-host.net? 1`] = `"https://logws1363.ati-host.net?s=598343&s2=70&p=pidgin.articles.cwl08rd38l6o.page&x1=[urn:bbc:optimo:cwl08rd38l6o]&x2=[responsive]&x3=[news-pidgin]&x4=[pcm]&x7=[article]&x8=[simorgh-nojs]&x9=[This+is+the+SEO+headline+of+this+test+article]&x11=[2019-07-12T13:00:04.940Z]&x12=[2019-09-27T15:56:33.775Z]"`;

exports[`Canonical Articles Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/pidgin"`;

exports[`Canonical Articles Footer I can see the footer copyright and external linking text 1`] = `"Â© 2020 BBC. De external site no concern BBC. De way wey we de take go external link."`;

exports[`Canonical Articles Header I can see the branding 1`] = `"BBC News, Pidgin"`;

exports[`Canonical Articles I can see a paragraph 1`] = `"This test image, copyright BBC, shows a map of France. The image is in the first three blocks and has this caption."`;

exports[`Canonical Articles I can see the headline 1`] = `"This is the headline of this test article"`;

exports[`Canonical Articles I can see the subheadline 1`] = `"This is a subheadline block"`;

exports[`Canonical Articles I can see the timestamp 1`] = `"12 July 2019"`;

exports[`Canonical Articles Images: I can see an image with a caption: Wetin we call dis foto, This test image, copyright BBC, shows a map of France. The image is in the first three blocks and has this caption. 1`] = `"https://ichef.bbci.co.uk/news/640/cpsdevpb/2d93/test/494cb5d0-dee2-11e9-b23c-f1ac9644bb55.jpg"`;

exports[`Canonical Articles Images: I can see an image with non-BBC copyright 1`] = `"Wia dis foto come from, Getty Images"`;

exports[`Canonical Articles Links: I can see a link: the BBC Weather homepage 1`] = `"https://www.bbc.co.uk/weather"`;

exports[`Canonical Articles SEO Apple Touch Icon: 72x72 1`] = `"https://news.files.bbci.co.uk/include/articles/public/pidgin/images/icons/icon-72x72.png"`;

exports[`Canonical Articles SEO Apple Touch Icon: 96x96 1`] = `"https://news.files.bbci.co.uk/include/articles/public/pidgin/images/icons/icon-96x96.png"`;

exports[`Canonical Articles SEO Apple Touch Icon: 128x128 1`] = `"https://news.files.bbci.co.uk/include/articles/public/pidgin/images/icons/icon-128x128.png"`;

exports[`Canonical Articles SEO Apple Touch Icon: 144x144 1`] = `"https://news.files.bbci.co.uk/include/articles/public/pidgin/images/icons/icon-144x144.png"`;

exports[`Canonical Articles SEO Apple Touch Icon: 152x152 1`] = `"https://news.files.bbci.co.uk/include/articles/public/pidgin/images/icons/icon-152x152.png"`;

exports[`Canonical Articles SEO Apple Touch Icon: 192x192 1`] = `"https://news.files.bbci.co.uk/include/articles/public/pidgin/images/icons/icon-192x192.png"`;

exports[`Canonical Articles SEO Apple Touch Icon: 384x384 1`] = `"https://news.files.bbci.co.uk/include/articles/public/pidgin/images/icons/icon-384x384.png"`;

exports[`Canonical Articles SEO Apple Touch Icon: 512x512 1`] = `"https://news.files.bbci.co.uk/include/articles/public/pidgin/images/icons/icon-512x512.png"`;

exports[`Canonical Articles SEO Apple Touch Icon: default 1`] = `"http://localhost:7080/pidgin/images/icons/icon-192x192.png"`;

exports[`Canonical Articles SEO Apple Touch Startup Image 1`] = `"https://news.files.bbci.co.uk/include/articles/public/pidgin/images/icons/icon-512x512.png"`;

exports[`Canonical Articles SEO Article author 1`] = `"https://www.facebook.com/bbcnews"`;

exports[`Canonical Articles SEO Canonical link 1`] = `"http://localhost:7080/pidgin/articles/cwl08rd38l6o"`;

exports[`Canonical Articles SEO Dir attribute 1`] = `"ltr"`;

exports[`Canonical Articles SEO FB admins 1`] = `"100004154058350"`;

exports[`Canonical Articles SEO FB app ID 1`] = `"1609039196070050"`;

exports[`Canonical Articles SEO Heading level 1 1`] = `"This is the headline of this test article"`;

exports[`Canonical Articles SEO Icons : 72x72 1`] = `"https://news.files.bbci.co.uk/include/articles/public/pidgin/images/icons/icon-72x72.png"`;

exports[`Canonical Articles SEO Icons : 96x96 1`] = `"https://news.files.bbci.co.uk/include/articles/public/pidgin/images/icons/icon-96x96.png"`;

exports[`Canonical Articles SEO Icons : 192x192 1`] = `"https://news.files.bbci.co.uk/include/articles/public/pidgin/images/icons/icon-192x192.png"`;

exports[`Canonical Articles SEO Lang attribute 1`] = `"pcm"`;

exports[`Canonical Articles SEO Linked data 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "Article",
  "author": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/pidgin.png",
      "width": 1024,
    },
    "name": "BBC News Pidgin",
    "noBylinesPolicy": "https://www.bbc.com/pidgin/institutional-48528766#authorexpertise",
  },
  "dateModified": "2019-09-27T15:56:33.775Z",
  "datePublished": "2019-07-12T13:00:04.940Z",
  "headline": "This is the SEO headline of this test article",
  "image": Object {
    "@type": "ImageObject",
    "height": 576,
    "url": "https://news.files.bbci.co.uk/ws/img/logos/og/pidgin.png",
    "width": 1024,
  },
  "mainEntityOfPage": Object {
    "@id": "http://localhost:7080/pidgin/articles/cwl08rd38l6o",
    "@type": "WebPage",
    "name": "This is the SEO headline of this test article",
  },
  "publisher": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/pidgin.png",
      "width": 1024,
    },
    "name": "BBC News Pidgin",
    "publishingPrinciples": "https://www.bbc.com/pidgin/institutional-48528766",
  },
  "thumbnailUrl": "https://news.files.bbci.co.uk/ws/img/logos/og/pidgin.png",
  "url": "http://localhost:7080/pidgin/articles/cwl08rd38l6o",
}
`;

exports[`Canonical Articles SEO OG description 1`] = `"This is the SEO headline of this test article"`;

exports[`Canonical Articles SEO OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/pidgin.png"`;

exports[`Canonical Articles SEO OG image alt 1`] = `"BBC News Pidgin"`;

exports[`Canonical Articles SEO OG locale 1`] = `"pcm"`;

exports[`Canonical Articles SEO OG site name 1`] = `"BBC News Pidgin"`;

exports[`Canonical Articles SEO OG title 1`] = `"This is the SEO headline of this test article - BBC News Pidgin"`;

exports[`Canonical Articles SEO OG type 1`] = `"article"`;

exports[`Canonical Articles SEO OG url 1`] = `"http://localhost:7080/pidgin/articles/cwl08rd38l6o"`;

exports[`Canonical Articles SEO Page title 1`] = `"This is the SEO headline of this test article - BBC News Pidgin"`;

exports[`Canonical Articles SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`Canonical Articles SEO Twitter card 1`] = `"summary_large_image"`;

exports[`Canonical Articles SEO Twitter creator 1`] = `"@BBCNews"`;

exports[`Canonical Articles SEO Twitter description 1`] = `"This is the SEO headline of this test article"`;

exports[`Canonical Articles SEO Twitter image alt 1`] = `"BBC News Pidgin"`;

exports[`Canonical Articles SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/pidgin.png"`;

exports[`Canonical Articles SEO Twitter site 1`] = `"@BBCNews"`;

exports[`Canonical Articles SEO Twitter title 1`] = `"This is the SEO headline of this test article - BBC News Pidgin"`;
